backend/src/main/resources/db/migration/V12.1__file_trigger_after_update_modified_on.sql

```diff
@@ -0,0 +1,10 @@
+DELIMITER //
+CREATE TRIGGER `file_after_update_modified_on`
```

Comment:
Same applies to all other triggers

Reply:
these triggers activate every time an UPDATE query updates a row, so this is not just a one time action

in this case, file_after_update_modified_on activates each time a row in `file` is updated

we've left the DB is in charge of maintaining modified_on fields in each table

but these particular tables are a problem - if we update a FileEntity, but don't change any part of it's base object ObjectEntity, hibernate will only update the `file` table, so modified_on won't update itself

Reply:
hmm ok for consistency lets keep it like that. On the other hand there are JPA annotations that handle this kinds of updates.

Reply:
I thought it would be simpler to have the DB do the heavy lifting
turns out `@CreationTimestamp` and `@UpdateTimestamp` would probably have been more straightforward